mlx/directives/item_directive.py

```diff
@@ -19,10 +19,8 @@ def perform_replacement(self, app, collection):
             collection (TraceableCollection): Collection for which to generate the nodes.
         """
         self._item = collection.get_item(self['id'])
-        header = self._item.get_id()
-        if self._item.caption:
-            header += ' : ' + self._item.caption
```

Comment:
caption of item is not shown anymore?

Reply:
I'm afraid that the caption is always shown now, i.e. the `:nocaptions:` flag in the _item_ directive is ignored. I'll open a PR to solve this. Example: https://melexis.github.io/sphinx-traceability-extension/checklist.html#CL-SOME_ITEM